webapp/src/store/cards.ts

```diff
@@ -322,3 +322,8 @@ export const getCurrentCard = createSelector(
     (state: RootState) => state.cards.cards,
     (current, cards) => cards[current],
 )
+
+export const GetHiddenCard = createSelector(
```

Comment:
nit: to keep the naming and casing consistent, I'd suggest something in plural and a bit more explicit, derived from `getCurrentBoardCards` as it's actually getting the inforrmation from that selector:

```suggestion
export const getCurrentBoardHiddenCards = createSelector(
```

Reply:
Sure, will change it.